aten/src/ATen/native/Sorting.cpp

```diff
@@ -185,8 +179,19 @@ void quantile_impl(
   if (q.dim() > 0) {
     out_shape.insert(out_shape.begin(), q.numel());
   }
-  resize_output(out, out_shape);
 
+  return out_shape;
+}
+
+Tensor quantile_compute(
+    const Tensor& self,
+    const Tensor& q,
+    const optional<int64_t> _dim,
+    const bool keepdim,
+    const QUANTILE_INTERPOLATION_MODE& interpolation,
+    const bool ignore_nan,
+    int64_t dim,
```

Comment:
nit: we should probably name `_dim` and `dim` better. Something like `original_dim` and one of {`wrapped_dim`, `sanitized`, `inferred_dim`} ?

Reply:
Yeah. That makes sense. Will do that in sometime and ping you. Thanks :)

Reply:
Done